test/src/test-reader/test-transformer.ts

```diff
@@ -136,6 +136,90 @@ describe("test-transformer", () => {
 
                 assert.match(transformedCode, new RegExp(`require\\("${moduleName}"\\)`));
             });
+
+            describe("replace import of esm module with a proxy", () => {
+                const moduleName = "esm-module";
+                const error = { message: "require() of ES Module", code: "ERR_REQUIRE_ESM" };
+                const expectedProxyValue =
+                    "" +
+                    `new Proxy({}, {\n` +
+                    `  get: function (target, prop) {\n` +
+                    `    return prop in target ? target[prop] : new Proxy(() => {}, this);\n` +
+                    `  },\n` +
+                    `  apply: function () {\n` +
+                    `    return new Proxy(() => {}, this);\n` +
+                    `  }\n` +
+                    `});\n`;
```

Comment:
nit: I would write it like this:

```js
const expectedProxyValue = [
    "",
    "new Proxy({}, {\",
    "    get: function (target, prop) {",
    ...
].join("\n")
```
just so there would be no need for manual "\n" at the end of each line